src/test/cli/ceph-conf/help.t

```diff
@@ -17,10 +17,14 @@
     -r|--resolve-search             search for the first file that exists and
                                     can be opened in the resulted comma
                                     delimited search list.
+    -D|--dump_all                   dump all variables.
```

Comment:
`--dump-all` to be more consistent.

Reply:
Looks like that help is inconsistent with code.

Reply:
Actually, I wasn't aware that there's code to convert dashes to underscores.

Reply:
what i mean is, `--dump-all` is more consistent with other options, like `--list-all-sections` and `--resolve-search`.

Reply:
In code I saw it used underscores, only in help were dashes mentioned, so I just assumed it's outdated help text and used underscore anyway, didn't know that both dash and underscore are fine.